src/main/java/org/sagebionetworks/bridge/services/ParticipantService.java

```diff
@@ -637,20 +637,15 @@ private void updateAccountAndRoles(App app, Account account, StudyParticipant pa
             updateRoles(requestContext, participant, account);
         }
         
-        // If the caller is not in a study, any study tags are allowed. If there 
-        // are any studies assigned to the caller, then the participant must be assigned 
-        // to one or more of those studies, and only those studies.
-        Set<String> callerStudies = RequestContext.get().getCallerStudies();
+        // If the caller is not in a study, any study tags are allowed. If the caller is
+        // assigned to any studies, and also adding studies to the account, then the studies
+        // must all be associated to the caller or it is a security violation.
+        Set<String> callerStudies = RequestContext.get().getOrgSponsoredStudies();
         if (!callerStudies.isEmpty()) {
             Set<String> studyIds = BridgeUtils.collectStudyIds(account);
-            if (studyIds.isEmpty()) {
-                throw new BadRequestException("Participant must be assigned to one or more of these studies: "
-                        + BridgeUtils.COMMA_JOINER.join(callerStudies));
```

Comment:
This is now allowed, since this association is taken from enrollments. When an admin updates another admin, they might have rights to see participants in specific organizations, but that doesn't mean they have to enroll that admin into one of those studies. Similarly, updating an account record doesn't imply that the user consented to enroll in a researcher study. Study IDs in the participant object is going to become read-only as part of cleaning up the current external ID code in favor of the enrollment AP, then enrollments won't be changed by account create/update at all.